plugins/factoids.py

```diff
@@ -187,6 +187,6 @@ def listfactoids(notice, chan):
 def listdetailedfactoids(chan):
     """- lists all available factoids with their respective data"""
     headers = ("Command", "Output")
-    data = [(fact[0], fact[1]) for fact in factoid_cache[chan].items()]
+    data = [(fact[0], fact[1]) for fact in sorted(factoid_cache[chan].items())]
     table = gen_markdown_table(headers, data).encode('UTF-8')
```

Comment:
`gen_markdown_table` is undefined at this point

Reply:
I can't reproduce that error.